src/transformers/data/data_collator.py

```diff
@@ -114,14 +193,29 @@ class DataCollatorWithPadding:
     padding: Union[bool, str, PaddingStrategy] = True
     max_length: Optional[int] = None
     pad_to_multiple_of: Optional[int] = None
+    return_tensors: str = 'pt'
 
-    def __call__(self, features: List[Dict[str, Union[List[int], torch.Tensor]]]) -> Dict[str, torch.Tensor]:
+    def __post_init__(self):
+        try:
+            if self.return_tensors == 'pt':
+                import torch
+            elif self.return_tensors == 'tf':
+                import tensorflow
+            elif self.return_tensors == 'np':
+                import numpy
+            else:
+                raise ValueError(f"Framework '{self.return_tensors}' not recognized!")
+        except ImportError:
+            raise ImportError(f"return_tensors is set to {self.return_tensors} "
+                              "but we were unable to load that framework!")
```

Comment:
Don't forget to run make style as this is not going to play well with black ;-)

Reply:
Done!